### ISSUE TITLE
Convert tutorial to use a CloudFormation stack

### ISSUE THREAD

**Issue by synchrophoto** [synchrophoto/DynamoDB-SDK-examples#26]
Need this [tutorial](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.Lambda.Tutorial.html) converted to use a yaml file to create a CloudFormation stack for creation.

**Comment by synchrophoto** [comment]
I will test this later today. I do think, if we can, to include the SNS stack. Are there downsides to doing that? What are your thoughts overall on this?

**Comment by DejanBelic** [comment]
I agree we should add SNS to stack too. I've tested before submitting PR. Don't see downsides... Will add then SNS to cloudformation too if that is okay?

**Comment by synchrophoto** [comment]
Yes please!

**Comment by johanrin** [comment]
@DejanBelic Can we close this issue?

**Comment by synchrophoto** [comment]
@DejanBelic were you able to add the SNS stuff? If so, can we close this issue?
